### DISCUSSION TITLE
Can we add a Tag list down the right hand side of the theme please

### DISCUSSION THREAD

**Discussion by davidejalexander** [Junyi-99/hugo-theme-anubis2#55] [Ideas]
**Thank you for the Anubis theme.** At Mydex CIC we are using it in our infrastructure technical blog aimed at Dev Ops and Systems Administrators https://infrastructure.mydex.org/ 

We use tags in all blogs to build up a consistent tag structure for helping indexing and linking blogs that touch different aspects of our platform infrastructure. We want to make it as easy as possible for visitors to look around and target their interest areas.  

We use a diverse range of open source components in our infrastructure https://dev.mydex.org/fyi/components-used.html which operates 24hrs a day and 365 days a year and is designed to be highly available, resilient and automated as much as possible. 

We have a small but talented dev ops and sys admin team. We are committed to sharing our knowledge and experience in our own give back to the open source community of developers and those who make use of open source components to deliver their own services.

**We would like to see a new element in theme** if possible that would put a tag list down the right hand slide in alphabetical order and perhaps with a basic filter option.  We felt this may be of benefit to other users of the Anubis theme so want to share this idea here for discussion as a candidate for future extensions.  We have also considered whether a tag cloud would be useful for readers of the blog.

**Comment by Junyi-99** [comment]
Hi @davidejalexander,

Thanks for the suggestion and for using the Anubis theme at Mydex CIC. Your teams work sounds impressive!

Could you provide some design illustrations or mockups for the tag list with a filter? This will help us understand your idea better.

Thanks!
